src/lib/client.ts

```diff
@@ -53,26 +54,22 @@ export class Client {
       },
       start: async (): Promise<void> => {
 
-        const farmerAddress = LocalStorage.getItem("rewardAddress")
-        if (farmerAddress === null) {
-          console.error("Reward address should not have been null...")
+        const farmerAddress: string | undefined = appConfig.getAppConfig()?.rewardAddress
+        if (farmerAddress == null) {
```

Comment:
`===` is usually preferred over `==`
in this case, you can just do `if (!farmerAddress) {`